app/src/main/java/com/example/leolam/myapplication/Building_List.java

```diff
@@ -54,7 +54,9 @@ protected void onCreate(Bundle savedInstanceState) {
             @TargetApi(Build.VERSION_CODES.M)
             public void onClick(View view) {
 
-                ActivityCompat.requestPermissions(Building_List.this, new String[]{Manifest.permission.CAMERA}, MY_PERMISSIONS_REQUEST_CAMERA);
+                //ActivityCompat.requestPermissions(Building_List.this, new String[]{Manifest.permission.CAMERA}, MY_PERMISSIONS_REQUEST_CAMERA);
```

Comment:
Just a nitpick, but we should probably start deleting unneeded comments to avoid future confusion.